apps/storybook/docs/d_planning/1_historiek-en-toekomst.stories.mdx

```diff
@@ -0,0 +1,135 @@
+met beperkt Javascript die gedocumenteerd worden a.d.h.v. voorbeelden.
+
+Van in het begin (2020) was het doel - en de aanpak - van UIG om de componenten van het Webuniversum aan te bieden als
+volwaardige web-componenten. Eens er een ruime set van web-componenten beschikbaar was verschoof de bulk van het werk
+naar onderhoud: bugs kwamen boven en uitbreidingen werden gevraagd.
+
+Sinds 2023 wordt er geregeld een planning opgesteld. Hierin zit een 'ongoing' deel, wat het weerkerend werk omvat en
+een 'focus' deel waarin we thematisch te werk willen gaan. Beknopt wordt er ook aangehaald wat er in de toekomst op de
+planning komt.
+
+Een samenvatting in pdf formaat van de meest recente planning 2024/2025 vind je
+<a href="UIG-Planning-2024-2025.pdf" target="_blank">hier</a>.
+
+## Nu: Planning 2024
+
+De focus ligt op:
+
+- het volledig wegwerken van de 'elementen', wat in een v2 release zal resulteren
+- de introductie van [web-types](https://github.com/JetBrains/web-types) zodat afnemers code completion en documentatie
+  in hun IDE hebben
+- de monorepo versimpelen - Nx gaf flexibiliteit, maar is nu eerder een nadeel nu de monorepo stabiel is
+- de OpenWC testen van de map vervangen door Cypress component-testen, ter voorbereiding van de grondige map
+  herwerking in 2025
+
+<br/><img src={planning2024} alt="planning-2024"/>
+
+
+## Toekomst: Planning 2025
+
+Het doel is in 2025 de UIG web componenten te finaliseren met een v3 release. Om dit te bereiken moeten we:
+
+- de map grondig herwerken, deze is historisch gegroeid en wordt veel gebruikt maar is ondertussen niet meer
+  consistent met de rest van de codebase
+- de legacy 'BaseElementOfType' basis klasse verwijderen, hiervoor zijn er nog componenten om te zetten naar Lit
+- de code kwaliteit nog verhogen door de volledige monorepo in 'strict' mode te zetten en door linting regels te
+  voorzien die we in de build verifiëren
+- de afgesplitste DV styling (in @domg/govflanders-style) wegwerken, we willen enkel css-in-js overhouden die
+  bij in de monorepo zit
+- in de mate van het mogelijke alle componenten zo CSP compliant als mogelijk maken
+- de WCAG compliancy, daar waar het kan, verbeteren
+
+<br/><img src={planning2025} alt="planning-2025"/>
+
+
+## Verleden: Q4 2023 / Q1 2024
+
+De backlog was opgekuist en werd stelselmatig weggewerkt. De focus lag hier op:
+
+- een aanzet geven tot het CSP compliant maken van componenten
+- een aanzet geven tot introductie van design patronen
+- het verder uitwerken van de nieuwe formulier aanpak met nieuwe form-components
+- starten met het wegwerken van de 'elementen'
+- de introductie van Cypress component-tests, naast en deels ter vervanging van de Cypress e2e-testen
+
+In het kader van het 'ongoing' werk werden er onder andere (voor het eerst) Cypress testen rond WCAG voorzien.
+
+<br/><img src={planning2023} alt="planning-2023"/>
+
+
+## Historiek
+
```

Comment:
Volgorde van Nu, Toekomst en dan Verleden vind ik goed, lijkt ook het meest relevant.

Echter ik zou de historiek wel degelijk in chronologische volgorde zetten. Als iemand het helemaal wil lezen voelt het wel raar om vanonder te beginnen.

Reply:
Er valt voor beide iets te zeggen, maar ook dan is het raar. Verleden en historiek volgen nu op elkaar (in omgekeerde volgorde), verleden zal nog uitbreiden (eens er een v2 release in 2025 is): 2024 wordt verleden en toekomst (2025) wordt nu - mogelijks maak ik in 2025 geen 2026 planning (wat gaan we daar op zetten?), dus toekomst verdwijnt dan. Op dat moment zal alles in omgekeerd historische volgorde zijn.